File: java/logica_y_validaciones/validar_agenda_vacia.java
package logica_y_validaciones;
public class validar_agenda_vacia {
    public static void main(String[] args) {
        int cantidadContactos = 0;
        if (cantidadContactos == 0) {
            System.out.println("La agenda está vacía.");
        } else {
            System.out.println("Hay contactos en la agenda.");
        }
    }
}

File: visual_basic/condicionales/validar_agenda_vacia/validar_agenda_vacia.vb
Imports System
Module validar_agenda_vacia
    Sub Main(args As String())
        Dim contactos() As String = {}
        If contactos.Length = 0 Then
            Console.WriteLine("La agenda está vacía 📭")
        Else
            Console.WriteLine("Contactos en la agenda:")
            For Each contacto In contactos
                Console.WriteLine(contacto)
            Next
        End If
    End Sub
End Module